tasks/Open To Evacuate or Not to Evacuate.md

```diff
@@ -0,0 +1,44 @@
+
+# Stimuli
+## The visual components
+![image](https://user-images.githubusercontent.com/78745728/117169986-9b2cbf00-ad97-11eb-84d7-66849fa6f64d.png)
+
+
+# Procedure
+## Steps
+1. Subjects were recruited via mTurk 
+2. At the start, subjects were required to answer whether they agree or disagree with three sentences to evaluate their personal preference with a five-level rating system (completely agree, agree, neither agree nor disagree, disagree, completely disagree). The sentences were ‘I am willing to take risks, in general’; ‘People should be willing to help others who are less fortunate; and ‘Most people can be trusted’.
+3. After answering the survey, each subject was asked to take a tutorial before the actual game would begin. In the tutorial, each subject separately interacted with three dummy players in a 45-second practice game.
+4. After the practice game, subjects were assessed for their comprehension of the game rules and payment structure using three multiple-choice questions, each with three options. If they failed to select the correct answer to any of the questions, they were dropped from the game. 
+5. At 720 s after the tutorial began, a ‘Ready’ button became visible simultaneously to all the subjects who completed the tutorial and passed the comprehension tests. 
+6. The real games started 30 s after the ‘Ready’ button showed up. If subjects did not click the button before the game started, they were dropped. 
+7. The games of networked groups required an exact number of subjects. When the subjects who successfully clicked the button were more than the required number, surplus subjects, who were randomly selected, were dropped from the game. When the number of qualified subjects was less than the required number, the game did not start (and subjects got their show-up fee). 
+8. At the start of the game, we selected one subject (the ‘informant’) at random who was informed in advance whether a disaster would indeed strike or not. The behaviours of the informants did not vary significantly across the network treatments
+>  The exact sentence that the informants received in their game screen was ‘A disaster is going to strike!’ when a disaster would strike or ‘There is no disaster’. when a disaster would not strike. Uninformed subjects received notification that ‘A disaster may or may not strike’.
+9. The other subjects were informed that some players had accurate information about the disaster, and they were also told that immediate neighbours of the informant would know the identity of the informant by our marking ‘i’ on the informant's node in their network diagram in the game screen. When a disaster struck before they evacuated, the subjects earned no bonus. Otherwise, they earned a bonus of $2.00 without the disaster or $1.00 with the disaster by spending $1.00 to evacuate, plus $0.10 per other player who took the correct action accordingly 
```

Comment:
In the network conditions, subjects shared their information; when clicking the 'Safe' button, their node turned blue and after 5 seconds automatically turned to grey, and likewise, when clicking the 'Danger' button, their node turned red. Subjects could only see the colors of neighbors that they were directly connected. Once subjects chose to evacuate, they couldn't send signals and their node turned grey, and their neighbors weren't informed of their evacuation.